### PR TITLE
Minimized AWS Docs for Kubeflow 1.3.1

### LINKED ISSUE #3166: Update AWS distribution website content with Kubeflow 1.3 documentation
Currently, [Kubeflow on AWS docs](https://www.kubeflow.org/docs/distributions/aws/) points to Kubeflow 1.2. AWS has completed testing and validation for Kubeflow 1.3.1 on EKS ([1.3 tracking issue](https://github.com/awslabs/kubeflow-manifests/issues/26)) and @judyheflin is working on documentation updates. Here is the work in progress branch - https://github.com/judyheflin/kubeflow-website-aws.git.

We would like to update the website master branch and 1.3-branch of the website with 1.3 documentation as soon as @judyheflin is ready with her PR. This issue is a heads up.

We are also working towards addressing #3074 with this change. I will update the issue with the plan.

### PR DESCRIPTION
**Pull request by judyheflin**
This is a PR for the minimized documentation for Kubeflow on AWS. We are still in the process of migrating the docs over to our own site, and in the meantime need these minimized docs added. 

Resolves #3166

### CODE REVIEW

**Review by akartsky — APPROVED** [review]

File: content/en/docs/distributions/aws/troubleshooting-aws.md [hunk 1]
```diff
@@ -1,117 +1,56 @@
...
 +++
 
-For general errors related to Kubernetes and Amazon EKS, please refer to the [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/troubleshooting.html) troubleshooting section.
+For general errors related to Kubernetes and Amazon EKS, please refer to the [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/troubleshooting.html) troubleshooting section. For issues with cluster creation or modification with `eksctl`, see the [`eksctl` troubleshooting](https://eksctl.io/usage/troubleshooting/) page.
 
-### ALB Fails to Provision
+### Validate prerequisites
 
-If you see your istio-ingress ADDRESS is empty after more than a few mins, it's possible that something is misconfigured in your ALB ingress controller.
+You may experience issues due to version incompatibility. Before diving into more specific issues, check to make sure that you have the correct [prerequisites](https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/vanilla#prerequisites) installed.
```
Comment (surajkota) [inline content/en/docs/distributions/aws/troubleshooting-aws.md:11]: this is not making sense. surakota@ check with kartik

File: content/en/docs/distributions/aws/component-guides/pipeline.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+authservice_session_cookie="<YOUR_COOKIE>"
+
+namespace="<YOUR_NAMESPACE>"
+
+client = kfp.Client(host=f"http://{kubeflow_gateway_endpoint}/pipeline", cookies=f"authservice_session={authservice_session_cookie}")
+client.list_experiments(namespace=namespace)
+```
+
+If you want to set up application load balancing (ALB) with Dex, see [Connect to Your Kubeflow Cluster](https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/vanilla#connect-to-your-kubeflow-cluster) and use the ALB address as the Kubeflow Endpoint.
```
Comment (surajkota) [inline content/en/docs/distributions/aws/component-guides/pipeline.md:52]: If you want to set up application load balancing (ALB) with Dex, see [Exposing Kubeflow over Load Balancer](https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/vanilla#exposing-kubeflow-over-load-balancer) and use the ALB address as the Kubeflow Endpoint.

**Review by surajkota — CHANGES_REQUESTED** [review]
Awesome Work on this! Left a few comments

**Pull request merged**